src/app/pages/shopping-cart-page/shopping-cart-page.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="cart">
+
+  <div class="cart__title">
+    Shopping cart
+  </div>
+  <div class="cart__notification" *ngIf="!this.shoppingCartService.amountOfSelectedProducts()">Your shopping cart is
+    empty. You can go to the catalog and add products
+  </div>
+
+  <div class="cart__product" *ngFor="let product of products$ | async">
```

Comment:
As I understand you already have component for product displaying? It will be good to use it here to, what do you think? Investigte it please.

Reply:
@an9949an 
I was alredy doing so using Vue.js. My opinion is that it looks less impressive. Also now I'm displayng more information, all information that is needed in needed format. Product.component displays only brief information.